builtins/builtins.go

```diff
@@ -840,6 +840,25 @@ func Hash(ctx context.Context, args ...object.Object) object.Object {
 	return object.NewByteSlice(h.Sum(nil))
 }
 
+func Spawn(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("spawn", 1, args); err != nil {
```

Comment:
We should allow passing arguments to the spawned function

Reply:
Yup